src/map.rs

```diff
@@ -87,6 +87,10 @@ impl MapCenterMode {
             MapCenterMode::Exact(position) => *position,
         }
     }
+
+    pub fn set_position(&mut self, position: Position) {
+        *self = MapCenterMode::Exact(position);
+    }
```

Comment:
This function is not necessary, you can just assign instead:

```rust
map_memory.center_mode = MapCenterMode::Exact(Position::new(1., 2.));
```

Reply:
Indeed, I'll remove it from my PR